lib/urlwatch/jobs.py

```diff
@@ -208,6 +212,17 @@ def pretty_name(self):
     def is_enabled(self):
         return self.enabled is None or self.enabled
 
+    @property
+    def tags(self) -> Optional[FrozenSet[str]]:
+        return self._tags
+
+    @tags.setter
+    def tags(self, value: Optional[Iterable[str]]):
+        if value is None:
+            self._tags = None
+        else:
+            self._tags = frozenset(value)
```

Comment:
Personal preference:

```suggestion
        if value is None:
            self._tags = None
        
        self._tags = frozenset(value)
```

Reply:
There's no return, so that would fall through and attempt to make a frozenset with None in it.